wagn/lib/wagn/config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Wagn.application.routes.draw do
```

Comment:
You need the Jasmine mount here, or you need to have two files and a condition on assigning the path for routes.